### ISSUE TITLE
Option to disable mythic keystone text

### ISSUE THREAD

**Issue by Rezy942** [frozn/TipTac#193]
**Is your feature request related to a problem? Please describe.**
Addon is currently adding text to top right corner of mythic keystone tooltips:
![image](https://user-images.githubusercontent.com/122007087/215808595-3ebb7b71-b265-4483-be22-e82a50a96b3e.png)

Clicking on a key link in chat will show this window - and as u can see, the offset isnt dynamically adjusted to move when there is a close button:
![image](https://user-images.githubusercontent.com/122007087/215808760-6befb734-b35a-4f9f-80f7-3629e982b919.png)

**Describe the solution you'd like**
Preferably a fix to move dynamically
And an option to disable this text all together. The wording is very confusing to me, im not sure what any of these things mean?
(TL, RL, and WRL)
If you could explain that would be great :)

**Describe alternatives you've considered**

**Additional context**

**Comment by Rezy942** [comment]
Nevermind - closing since i just found that option under tooltip item ref

**Comment by frozn** [comment]
TL = time limit
RL = reward level
WRL = weekly reward level

I reopened this issue because of the overlapping info text with the close button. In the original blizzard code there is a check for overlapping and in this case the tooltip will be resized. BUT: This code only considers the text in the first line on the left side, and not the text on the right side if it exists.

So I improved this behaviour with readjusting the tooltip after blizzards code has run. This will be included in the next release.
